doc/misc/system-state.7.rst

```diff
@@ -0,0 +1,40 @@
+..
```

Comment:
We might add a reference to this new man page in the main `dnf5.8.rst` and perhaps also in the `install` and `mark` commands man pages.

Reply:
Added to the main `dnf5(8)` man page.